contracts/budget-contracts/src/main/java/org/java/mentorship/contracts/budget/client/TransactionFeignClient.java

```diff
@@ -13,7 +13,7 @@ public interface TransactionFeignClient {
             method = RequestMethod.GET,
             value = "/transactions"
     )
-    List<Transaction> getTransactions();
+    List<Transaction> getTransactions(@RequestParam(required = false, name = "recent") Boolean recent);
```

Comment:
We generally don't do this.
Recent can mean anything (1 day, 2 days, 30 days) so instead in a real environment we would use "fromTime" or "fromDay" and the client (UI) would pass a time or date.
However, since we're on a time constraint I'll let it pass